test/unit/storage/conversion/RdfToQuadConverter.test.ts

```diff
@@ -14,10 +15,25 @@ import { INTERNAL_QUADS } from '../../../../src/util/ContentTypes';
 import { BadRequestHttpError } from '../../../../src/util/errors/BadRequestHttpError';
 const { namedNode, triple, literal, quad } = DataFactory;
 
+// All of this is necessary to not break the cross-fetch imports that happen in `rdf-parse`
+jest.mock('cross-fetch', (): any => {
+  const mock = jest.fn();
+  // Require the original module to not be mocked...
+  const originalFetch = jest.requireActual('cross-fetch');
+  return {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    __esModule: true,
+    ...originalFetch,
```

Comment:
Do we actually need the original fetch? Would be an issue to have tests depending on a Web resource.

Reply:
The reason for doing it like this is that internally the conversion library also uses other imports from that import. If this specific line gets removed the test fails with `TypeError: cross_fetch_1.Headers is not a constructor`. So adding the non-fetch parts of the import like this was the easiest. I could also look into mocking specifically the `Headers` constructor (and hoping that doesn't break anything else further down the line).

Reply:
Nah that's okay, thanks!